From: Matthias Brugger <mbrugger@suse.com>
Subject: arch: arm64: acpi: KABI ginore includes
Patch-mainline: never, kabi
References: bsc#1117158 bsc#1134671

Upstream commit 
09ffcb0d718a ("arm64: acpi: fix alignment fault in accessing ACPI")
adds new header file inclusions which trigger kABI checker to fail.
Mark the include statements as not needed for kABI checker to fix that.

Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 arch/arm64/include/asm/acpi.h |    3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm64/include/asm/acpi.h
+++ b/arch/arm64/include/asm/acpi.h
@@ -12,7 +12,10 @@
 #ifndef _ASM_ACPI_H
 #define _ASM_ACPI_H
 
+#ifndef __GENKSYMS__
 #include <linux/efi.h>
+#endif
+
 #include <linux/memblock.h>
 #include <linux/psci.h>
 
